### PR TITLE
Initial Linea ENS documentation

### PR DESCRIPTION
**Pull request by arthur-remy**
# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [X] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [X] You've previewed your changes locally.

## Describe the change

Add documentation for Linea ENS

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

**Comment by arthur-remy** [comment]
@emaConsenSyscosc ready for your review

File: project-words.txt [hunk 1]
```diff
@@ -679,4 +679,5 @@ redcircle
 bluecircle
 tatarot
 Tatarot
-Scry
\ No newline at end of file
+Scry
+ENSIP
```
Comment (emilyJLin95) [inline project-words.txt:683]: @arthur-remy 
can you do this in alphabetical order - there are merge conflicts

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]
left a comment about merge conflicts

File: project-words.txt (hunk 1 above)
Comment (arthur-remy) [reply project-words.txt]: Done

**Comment by arthur-remy** [comment]
Note: the lint check fails because of another page that has nothing to do with Linea ENS

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**